Dockerfile

```diff
@@ -68,6 +71,14 @@ FROM base as build
 
 ENV SC_BUILD_TARGET=build
 
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg && \
+  echo \
+  "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian \
+  $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null && \
+  apt-get update && \
+  apt-get install -y --no-install-recommends \
+  docker-ce docker-ce-cli containerd.io
```

Comment:
You might want to fix the docker version here, or is it not important?

Reply:
and by the way, maybe think of installing docker compose v2... not sure that comes automatically in.

Reply:
@GitHK : Not important I think as the host's docker socket is used
@sanderegg : will adress